applications/asset_tracker_v2/src/events/sensor_module_event.h

```diff
@@ -30,7 +30,17 @@ enum sensor_module_event_type {
 	 */
 	SENSOR_EVT_MOVEMENT_DATA_READY,
 
-	/** Environmental sensors have been sampled.
+	/** Accelerometer reported activity.
+	 *  Acceleration exceeded the configured activity threshold.
+	 */
+	SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED,
+
+	/** Accelerometer reported inactivity.
+	 *  Acceleration stayed below the threshold for a given time.
```

Comment:
`for a given time` should also be changed in the follow up PR for the same reasons I mentioned in the comment above.